frappe/utils/error.py

```diff
@@ -70,6 +71,9 @@ def log_error(
 	)
 	capture("error_logged", "frappe", properties={"title": title, "trace_id": trace_id})
 
+	# Capture exception data if telemetry is enabled
+	capture_exception(message=f"{title}\n{traceback}")
```

Comment:
This will trigger import of sentry even if it's not configured. Can avoid that memory cost by only importing it if sentry is configured.

Also if import is required, it's best to preload all those modules in app.py.

Refer https://github.com/frappe/frappe/pull/21475